### ISSUE TITLE
During object detection, How to extract image feature from neck layer during eval mode in MMDET 3.3.0?

### ISSUE THREAD

**Issue by prafful-kumar** [open-mmlab/mmdetection#11825]
For example, I have a faster-rcnn model pre-trained on coco dataset and I am fine-tuning on downstream dataset. To analyse the features of the detector, I want to extract the embeddings from the neck (i.e., FPN), but I am not sure how to extract the feature embeddings. Can you help me understand and implement it?

**Comment by PeterVennerstrom** [comment]
The [detector level abstraction ](https://github.com/open-mmlab/mmdetection/blob/cfd5d3a985b0249de009b67d04f37263e11cdf3d/mmdet/models/detectors/two_stage.py#L227)is one place you could access the embeddings.
